tensorflow/python/ops/math_ops.py

```diff
@@ -5609,8 +5608,7 @@ def rsqrt(x, name=None):
   numpy=array([0.707, inf, nan], dtype=float32)>
 
   Args:
-    x: A `tf.Tensor`. Must be one of the following types: `bfloat16`, `half`,
-      `float32`, `float64`.
+    x: A `tf.Tensor`. Must be one of the following types: `bfloat16`, `half`, `float`, `double`, `complex64`, `complex128`.
```

Comment:
float32 / float64?

Reply:
> float32 / float64?

code: Tensorflow Version（2.11）

    import tensorflow as tf
    tf.math.rsqrt("sss")

result:InvalidArgumentError: Value for attr 'T' of string is not in the list of allowed values: bfloat16, half, float, double, complex64, complex128; NodeDef: {{node Rsqrt}}; Op<name=Rsqrt; signature=x:T -> y:T; attr=T:type,allowed=[DT_BFLOAT16, DT_HALF, DT_FLOAT, DT_DOUBLE, DT_COMPLEX64, DT_COMPLEX128]> [Op:Rsqrt]

As can be seen from the above code, the type of parameter x  supports complex types.

Reply:
float32 and float are equivalent.
float64 and double are equivalent.

I believe in python we usually use the former, so can you change those two back?